plrustc/rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "1.67.1"
-components = [ "rustfmt", "rust-src", "rustc-dev", "cargo" ]
+channel = "1.69.0"
+components = [ "rustfmt", "rust-src", "rustc-dev", "cargo", "llvm-tools" ]
```

Comment:
A++

I ran into needing that locally and wasn't sure what to do.  I ended up figuring out what I needed to hack from the CLI, but this is better.

Reply:
It's kind of a rustc bug that it's separately needed, TBH.

Reply:
okay.  I'll just anxiously await the plrust installer tool.